configure.sh

```diff
@@ -147,7 +150,7 @@ build:android_x86_64 --fat_apk_cpu=x86_64
 # The default android SDK/NDK paths are hardcoded here and the user needs
 # to change the paths according to the local configuration or
 # the "install_android.sh" script
-build --action_env ANDROID_NDK_HOME="/tmp/lce_android/ndk/17.2.4988734"
+build --action_env ANDROID_NDK_HOME="/tmp/lce_android/ndk/18.1.5063045"
```

Comment:
@lgeiger  is this change necessary? I locally still have the older version and want to experiment with this a bit

Reply:
Yes, on my system it broke when building the benchmarking binary for ARM64 otherwise https://github.com/larq/compute-engine/pull/258#issuecomment-600616052